src/common/utils/auth_tests.rs

```diff
@@ -234,20 +2426,37 @@ mod tests {
         .await
     }
 
-    // TODO: Add test for organization creation after org_cloud_singe branch is
-    // merged.
+    // Tests for routes defined in handler::http::request::clusters.
 
-    // Tests for routes defined in handler::http::request::organization::setings.
+    #[tokio::test]
+    async fn list_clusters() {
+        test_auth(
+            Method::GET,
+            format!("api/clusters"),
+            AuthExtractor {
+                auth: AUTH_HEADER_VAL.to_string(),
+                // Should these be empty strings?
+                method: format!(""),
+                o2_type: format!(""),
+                org_id: format!(""),
+                bypass_check: true,
+                parent_id: format!("default"),
+            },
+        )
+        .await
+    }
```

Comment:
?

Reply:
Yep, if `bypass_check` is true that means, we don't care about the values of `o2_type` and all